config/initializers/intercom.rb

```diff
@@ -0,0 +1,119 @@
+IntercomRails.config do |config|
+  # == Intercom app_id
+  #
+  config.app_id = ENV["INTERCOM_APP_ID"] || "rtcpj4hf"
+
+  # == Intercom session_duration
+  #
+  # config.session_duration = 300000
+  # == Intercom secret key
+  # This is required to enable Identity Verification, you can find it on your Setup
+  # guide in the "Identity Verification" step.
+  #
+  # config.api_secret = "..."
+
+  # == Enabled Environments
+  # Which environments is auto inclusion of the Javascript enabled for
+  #
+  config.enabled_environments = ["development", "production"]
+
+  # == Current user method/variable
+  # The method/variable that contains the logged in user in your controllers.
+  # If it is `current_user` or `@user`, then you can ignore this
+
+  config.user.current = proc {}
```

Comment:
proc{} will return nil right? so this line can probably deleted if its the default current user other wise `config.user.current = proc {other_name_for_current_user}`